apps/fabric-website/src/pages/GetStarted/GetStartedPage.tsx

```diff
@@ -185,7 +185,7 @@ initializeIcons('https://my.cdn.com/path/to/icons/');`
               <p>Add the following line to the &lt;head&gt; of your webpage:</p>
               <CodeBlock language='html' isLightTheme={ true }>
                 {
-                  `<link rel="stylesheet" href="https://static2.sharepointonline.com/files/fabric/office-ui-fabric-core/9.0.0/css/fabric.min.css">`
+                  `<link rel="stylesheet" href="https://static2.sharepointonline.com/files/fabric/office-ui-fabric-core/9.2.0/css/fabric.min.css">`
```

Comment:
Can this value be pulled in from package.json too? Would be great to minimize the number of manual updates needed when we do version bumps.

Reply:
I totally agree, just couldn't figure out how to add a variable inside a CodeBlock. Perhaps @battletoilet knows?

Reply:
You should be able to just use an interpolated variable since CodeBlocks use template strings (indicated by the backtick `). So it might look something like this:

```javascript
<link rel="stylesheet" href="https://static2.sharepointonline.com/files/fabric/office-ui-fabric-core/${ corePackageData.version }/css/fabric.min.css">
```

Reply:
Worked!